packages/react-composites/tests/browser/call/hermetic/PeoplePane.test.ts

```diff
@@ -9,185 +9,209 @@ import {
+
+    expect(await stableScreenshot(page)).toMatchSnapshot(`video-gallery-page-participants-flyout-custom-ellipses.png`);
   });
-});
 
-test.describe('Call composite participant menu items injection tests', async () => {
-  // TODO: Split this test into multiple tests: one for beta/desktop, beta/mobile, stable each.
-  // Do this after the test has been stabilized. Keep the same name for flakiness analysis.
   test('injected menu items appear', async ({ page, serverUrl }, testInfo) => {
-    const paul = defaultMockRemoteParticipant('Paul Bridges');
-    addVideoStream(paul, true);
-    paul.isSpeaking = true;
-    const fiona = defaultMockRemoteParticipant('Fiona Harper');
-    addVideoStream(fiona, true);
-    const participants = [paul, defaultMockRemoteParticipant('Eryka Klein'), fiona];
-    const initialState = defaultMockCallAdapterState(participants);
+    test.skip(!participantListShownAsFullScreenPane(testInfo));
 
     await page.goto(
-      buildUrlWithMockAdapter(serverUrl, initialState, {
+      buildUrlWithMockAdapter(serverUrl, participantListInitialState(), {
         injectParticipantMenuItems: 'true',
         callInvitationUrl: 'testUrl'
       })
     );
     await waitForSelector(page, dataUiId(IDS.videoGallery));
 
-    // Open participants flyout.
-    if (!isTestProfileDesktop(testInfo) && !isTestProfileStableFlavor()) {
-      await pageClick(page, dataUiId('call-with-chat-composite-more-button'));
-      const drawerPeopleMenuDiv = await page.$('div[role="menu"] >> text=People');
-      await drawerPeopleMenuDiv?.click();
-    } else {
-      await pageClick(page, dataUiId('call-composite-participants-button'));
-    }
-    if (flavor === 'beta') {
-      if (!isTestProfileDesktop(testInfo)) {
-        // click the first participant
-        await pageClick(page, `${dataUiId('participant-list')} [role="menuitem"]`);
-        // wait for drawer to have opened
-        await waitForSelector(page, dataUiId('drawer-menu'));
-      } else {
-        await page.hover(dataUiId('participant-item'));
-        await pageClick(page, dataUiId(IDS.participantItemMenuButton));
-        await waitForSelector(page, '.ms-ContextualMenu-itemText');
-      }
-    } else {
-      // Open participant list flyout
-      await pageClick(page, dataUiId(IDS.participantButtonPeopleMenuItem));
-      await page.hover(dataUiId('participant-item') + ' >> nth=0');
-      // There should be at least one participant. Just click on the first.
-      await pageClick(page, dataUiId(IDS.participantItemMenuButton) + ' >> nth=0');
-
-      const injectedMenuItem = await waitForSelector(page, dataUiId('test-app-participant-menu-item'));
-      await injectedMenuItem.waitForElementState('stable', { timeout: perStepLocalTimeout() });
-    }
+    await pageClick(page, dataUiId('call-with-chat-composite-more-button'));
+    const drawerPeopleMenuDiv = await page.$('div[role="menu"] >> text=People');
```

Comment:
Would there be any value in making a DataUIId for this button?

Reply:
I'll refrain from adding that in this PR.

I think there would be value in adding a helper function for this. Raw `$()` expressions are hard to read in a test.